tests/Api/StatsTest.php

```diff
@@ -109,6 +125,32 @@ public function totalProvider()
                     ],
                 ]),
             ],
+            'clicked events' => [
```

Comment:
I can see you have added to this payload both the `clicked` and `opened` events, but I cannot see either the `unsubscribed` or `stored`. Are they already here and I didn't notice them?

Reply:
I initially just added the clicked and opened, but added the others now.